src/patterns/FavoritesCollectionTile/FavoritesCollectionTile.tsx

```diff
@@ -161,6 +163,7 @@ const FavoritesCollectionTile = memo(
                             height={iconSize}
                             color="$dark-gray"
                             className={`${baseClassName}__icon-button`}
+                            title="Manage List"
```

Comment:
believe it or not, I'm not a fan of the title on svgs because it results in a weird behavior where it shows an ugly gray popover if you hover for a while